drivers/SmartThings/zwave-switch/src/test/test_zooz_zen_30_dimmer_relay_preferences.lua

```diff
@@ -0,0 +1,534 @@
+
+local test = require "integration_test"
+local t_utils = require "integration_test.utils"
+local zw = require "st.zwave"
+local zw_test_utils = require "integration_test.zwave_test_utils"
+local Configuration = (require "st.zwave.CommandClass.Configuration")({ version=4 })
+
+local zooz_zen_dimmer_relay_endpoints = {
+    {
+        command_classes = {
+            {value = zw.METER},
+            {value = zw.BASIC},
+            {value = zw.SWITCH_BINARY},
+            {value = zw.SWITCH_MULTILEVEL},
+            {value = zw.SENSOR_MULTILEVEL}
```

Comment:
Does this device support SENSOR_MULTILEVEL ?

Reply:
> Does this device support SENSOR_MULTILEVEL ?

no, I deleted this command class